pajbot/modules/ascii.py

```diff
@@ -86,13 +95,13 @@ def check_message(message):
         return False
 
     def on_pubmsg(self, source, message, tags, **rest):
-        if self.settings["enabled_by_stream_status"] == "Online Only" and not self.bot.is_online:
+        if source.level >= self.settings["bypass_level"] or source.moderator is True:
             return
 
-        if self.settings["enabled_by_stream_status"] == "Offline Only" and self.bot.is_online:
+        if self.settings["enabled_by_stream_status"] == "Online Only" and not self.bot.is_online:
             return
 
-        if source.level >= self.settings["bypass_level"] or source.moderator is True:
+        if self.settings["enabled_by_stream_status"] == "Offline Only" and self.bot.is_online:
```

Comment:
What change does this make? Right now this just looks like you just shuffled the statements around

If it serves no purpose anymore, just revert them back to their original state